.mocharc.json

```diff
@@ -4,5 +4,11 @@
     "exclude": [
         "test/soak_test/**",
         "test/user_code/**"
+    ],
+    "reporter": "mocha-jenkins-reporter",
+    "reporter-option": [
+        "junit_report_name=Node.js Client Tests",
+        "junit_report_path=report.xml",
+        "junit_report_stack=1"
```

Comment:
Will this affect the stack we see on test failures? It would be very helpful if we can see the full stack trace on test failures

Reply:
not it enables stack, we can see trace